Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -6463,9 +6463,8 @@ void WebPage::drawPagesToPDFImpl(FrameIdentifier frameID, const PrintInfo& print
             if (!m_printContext)
                 return;
 
-            size_t pageCount = m_printContext->pageCount();
             for (uint32_t page = first; page < first + count; ++page) {
-                if (page >= pageCount)
+                if (page >= m_printContext->pageCount())
```

Comment:
Why? Can this actually change the page count?

Reply:
I don't have evidence that the code in the loop changes the page count but I figured it was better to be safe than sorry. I also wanted to be consistent with the other call site, which I am fixing.